spi/topology/src/main/java/quarks/topology/spi/graph/TWindowImpl.java

```diff
@@ -30,4 +35,20 @@
         Aggregate<T,U,K> op = new Aggregate<T,U,K>(window, processor);
         return feeder().pipe(op); 
     }
+
+    @Override
+    public <U> TStream<U> batch(BiFunction<List<T>, K, U> batcher) {
+        batcher = Functions.synchronizedBiFunction(batcher);
+        Window<T, K, List<T>> window =
+                Windows.window(
```

Comment:
Ok, I see now that it does, but there is an unexpected (to me) implementation detail. After processing the batch I was expecting the tuples to be evicted right away, as they have no value at that point. 

I think this implementation is waiting until the next tuple (which may never come).